src/css/main.less

```diff
@@ -905,3 +907,44 @@ fieldset[disabled] .btn-info.active {
     padding: .8em 1em; /* this would be "10px 0" otherwise */
   }
 }
+
+.tab-pane.active.flex-container {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  min-height: 100vh;
+
+  .wrapper {
+    display: flex;
+    flex: 1;
+
+    main {
+      flex: 1;
+      padding-left: 15px;
+    }
+
+    nav {
```

Comment:
No, I would introduce this right now, it's much more expressive, also to the reviewer and also useful.